src/renderer/modals/ClaimRewards/steps/StepRewards.js

```diff
@@ -0,0 +1,87 @@
+// @flow
+import invariant from "invariant";
+import React from "react";
+import styled from "styled-components";
+
+import { Trans } from "react-i18next";
+
+import type { StepProps } from "../types";
+
+import { getMainAccount } from "@ledgerhq/live-common/lib/account";
+import { formatCurrencyUnit } from "@ledgerhq/live-common/lib/currencies";
+import TrackPage from "~/renderer/analytics/TrackPage";
+import Box from "~/renderer/components/Box";
+import Button from "~/renderer/components/Button";
+import CurrencyDownStatusAlert from "~/renderer/components/CurrencyDownStatusAlert";
+import Text from "~/renderer/components/Text";
+
+import { BigNumber } from "bignumber.js";
+import InfoCircle from "~/renderer/icons/InfoCircle";
+import ClaimRewards from "~/renderer/icons/ClaimReward";
+
+const IconWrapperCircle = styled(Box)`
+  width: 64px;
+  height: 64px;
+  border-radius: 50%;
+  background: ${props => (props.green ? "#66be5419" : "#6490f119")};
+  color: ${props => (props.green ? "#66be54" : "#6490f1")};
+  align-items: center;
+  justify-content: center;
+  align-self: center;
+`;
+
+export default function StepRewards({ account, parentAccount, reward }: StepProps) {
+  invariant(account, "account is required");
+  const mainAccount = getMainAccount(account, parentAccount);
+
+  const formattedReward = formatCurrencyUnit(mainAccount.unit, BigNumber(reward || 0), {
+    disableRounding: true,
+    alwaysShowSign: false,
+    showCode: true,
+  });
+
+  return (
+    <Box flow={1}>
+      <TrackPage category="Receive Flow" name="Step 1" />
+      {mainAccount ? <CurrencyDownStatusAlert currency={mainAccount.currency} /> : null}
```

Comment:
imo remove this CurrencyDownStatusAlert. also the TrackPage is saying Receive Flow.